plutus-core/executables/uplc/Main.hs

```diff
@@ -212,6 +219,11 @@ runConvert (ConvertOptions inp ifmt outp ofmt mode) = do
     program <- (getProgram ifmt inp :: IO (UplcProg PLC.AlexPosn))
     writeProgram outp ofmt mode program
 
+runDumpModel :: IO ()
+runDumpModel = do
+    let params = fromJust PLC.defaultCostModelParams
+    BSL.putStr $ Aeson.encode params
```

Comment:
Do we maybe want to format the output more readably?  At the moment it prints 
```
{"addInteger-cpu-arguments-intercept":197209,"addInteger-cpu-arguments-slope":0,"addInteger-memory-arguments-intercept":1,"addInteger-memory-arguments-slope":1,"appendByteString-cpu-arguments-intercept":396231,"appendByteString-cpu-arguments-slope":621,"appendByteString-memory-arguments-intercept":0,"appendByteString-memory-arguments-slope":1,"appendString-cpu-arguments-intercept":150000,...
```
Maybe something like 
```
addInteger-cpu-arguments-intercept: 197209
addInteger-cpu-arguments-slope: 0
addInteger-memory-arguments-intercept: 1
addInteger-memory-arguments-slope: 1
appendByteString-cpu-arguments-intercept: 396231
appendByteString-cpu-arguments-slope: 621
...
```
would be more readable.  I suppose it depends on what we want to do with the output though.

Reply:
It's also slightly annoying that it mixes up the CEK parameters and the builtin parameters, but I suppose that's what's in the map.

Reply:
I originally did that, but I anticipate that people will want to process it, so a standard format is nice. Getting it pretty-printed is a matter of piping it through `jq`